### DISCUSSION TITLE
Github doesn't show all the commits.

### DISCUSSION THREAD

**Discussion by vin-and** [community/community#39744] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

As we can see, it suggests that I've made 3 commits, all of them on Nov 22
![image](https://user-images.githubusercontent.com/77638293/203311065-7fa9e4b2-5a22-4727-a53e-d27a567a8f7e.png)
Yet, 
![image](https://user-images.githubusercontent.com/77638293/203311254-d0120069-dc00-449d-a73b-be624d1face3.png)
contradicts this.

What could I do so that github sees all my commits? It might be worth noting that I had changed my e-mail before any of the commits listed, which appeared to remove the previous ones

**Comment by RONIE70** [comment]
Puedes cambiar la forma en la que se muestran los diffs en GitHub Desktop para que satisfagan tus necesidades de revisión.

Para cambiar la forma en la que ves los diffs, en la esquina superior derecha de la vista del diff, haz clic en .

Para cambiar cómo se muestra toda la diferencia, en "Diff display" (Pantalla de diferencias), seleccione Unified (Unificada) o Split (Dividida). La vista unificada te muestra los cambios de forma linea, mientras que la vista dividida te muestra el contenido antiguo en la parte izquierda y el nuevo en la derecha.
A fin de ocultar los cambios de espacios en blanco para que se pueda centrar en cambios más sustanciales, seleccione Hide Whitespace Changes (Ocultar los cambios de espacios en blanco).

**Comment by airtower-luna** [comment, marked as answer]
> It might be worth noting that I had changed my e-mail before any of the commits listed, which appeared to remove the previous ones

That explains it. If you remove an email address from your account, GitHub won't recognize commits with that author email as yours any more. The solution would be to add the address to your account again (but it may take a while for the contributions list to update).

**Reply by vin-and** [reply]
The missing commits were made with the new e-mail. As you can see, the number of commits decreased again over the span of 10h
![image](https://user-images.githubusercontent.com/77638293/203420738-17b26c8d-9a5c-4722-b502-c724b91db810.png)

**Reply by airtower-luna** [reply]
I'm afraid it's impossible to say why that might be going on without seeing the history, looking at your profile I only see "1 contribution in private repositories".

You can check [Why are my contributions not showing up on my profile?](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile), it lists the exact conditions your commits need to fulfill to count.
